cmd/cvp-trigger/main.go

```diff
@@ -350,3 +356,19 @@ func writeResultOutput(prowjobResult jobResult, outputPath string) error {
 
 	return nil
 }
+
+// appendMultiStageParams passes all of the OO_ params to ci-operator as multi-stage-params.
+// TODO: After all of the CVP changes required to accept the multi-stage-params have been made, we need to go back and
+// remove the OO_ env params from here.
```

Comment:
I think the TODO is a bit misleading: this needs to stay, what will need to be removed is setting them as envvars. Basically the `envvars` on L225 will be initialized empty rather than from `params`.

No need to fix ofc, just wanted to clarify.

Reply:
Yeah, you're right - that comment is a little misleading. I must've written it when I was tired, lol